packages/uikit/src/components/Svg/Icons/ZksyncWithOutCircle.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Svg from "../Svg";
+import { SvgProps } from "../types";
+
+export const ZksyncWithOutCircleIcon: React.FC<React.PropsWithChildren<SvgProps>> = (props) => {
+  return (
+    <Svg viewBox="0 0 27 15" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
```

Comment:
For some svgs that no need to manipulate, can consider just load svg by img?

Reply:
ok, sure